### ISSUE TITLE
On-fly test coverage stucked

### ISSUE THREAD

**Issue by amrix08** [tntim96/JSCover#312]
I'm new with JSCover.

I've read the manual and viewed the examples but I still can't apply it on our projects.

We have this site **(http://xxxx.xx.com:20500/dir/dir)** that is being run on https but based on the article I've read, there's no way to get the coverage of the HTTPS sites. So what we did was provided a http site of that site. 

These are the steps I did.

Pre-requisites:
- Changed the proxy server host to **localhost** and port to **3128**
- Created a shortcut of google chrome and adjusted it's parameter to below
`"C:\Program Files\Google\Chrome\Application\chrome.exe" --disable-web-security --disable-gpu --user-data-dir=C:\Users\XXX\AppData\Local\Google\Chrome\User Data\Default\Cache --allow-file-access-from-files`

1. Run **the jscover-proxy.bat** on a cmd. This is the content of the bat
`java -jar target\dist\JSCover-all.jar -ws --proxy --port=3128 --report-dir=target/jscover-proxy`
2. Opened the **http://localhost:3128/jscoverage.html**
3. Supplied the **(http://xxxx.xx.com:20500/dir/dir)** in URL field
4. The site then loads up but there was an error encountered which is described below
`Uncaught DOMException: Blocked a frame with origin "http://xxxx.xx.com:20500/dir/dir" from accessing a cross-origin frame.`
5. No reports was being created, no scripts generated, json file was empty now I'm stucked. 

Basically, JSCover have 3 modes and we want to implement the on-fly coverage method which is the **server** mode.

If anyone can give insights on how we can proceed, that will be a great help. Thank you!

**Comment by tntim96** [comment]
This example does what you want https://github.com/tntim96/JSCover/tree/master/examples/localStorage-file-system

Basically you have to file-instrument the code, and upload them to your server before running the tests.

**Comment by amrix08** [comment]
The thing is we don't have the source code to instrument the file or is it a pre-requisite?

**Comment by tntim96** [comment]
It's a pre-requisite

**Comment by amrix08** [comment]
So to confirm we can't test the site on-fly? We have to have our own copy of the source code to instrument the files? What if we don't have the source code for example, we want to test coverage the google site?

**Comment by tntim96** [comment]
That would be a problem because that code is minified, and JSCover works best by instrumenting the original source.

**Comment by amrix08** [comment]
![image](https://user-images.githubusercontent.com/62284865/153830331-1ca11e36-756f-4fc6-b732-560001e6f61d.png)

Tried the samples on the link you've provided but got an error. Tried googling it and they said that storeEval is changed with execute-script. I tried changing it to execute-script but still same error

**Comment by tntim96** [comment]
Good pickup. I replaced `storeEval` with `execute script` and it worked for me.
